core/consensus/aura/src/lib.rs

```diff
@@ -657,6 +660,68 @@ fn authorities<A, B, C>(client: &C, at: &BlockId<B>) -> Result<Vec<A>, Consensus
 		.ok_or_else(|| consensus_common::Error::InvalidAuthoritiesSet.into())
 }
 
+/// Means of determining a list of authority keys, generally used to determine if we are an
+/// authority by cross-referencing this list against the keys in the store and finding one that
+/// appears in both.
+///
+/// This is implemented directly on the `AppPair` key type.
+pub trait AuthorityProvider: AppPair {
```

Comment:
Do we actually use this anywhere? Will we? (I guess not as long as we support being a multi-authority in Aura like in Babe, Grandpa and ImOnline?)

Reply:
No, but I also did not fixed `Aura` yet. Will come in a later pr.

Reply:
is there an issue for the rest of the PRs?